package.json

```diff
@@ -11,17 +11,14 @@
   },
   "scripts": {
     "worker": "wrangler dev --env dev --port 4001",
-    "start": "tsx src/index.ts",
```

Comment:
No more start command? Isn't it convenient to run and test like this? Or does this require a slash command

Reply:
`bun start worker` is used to start worker plugins.